abm/simulation/sims.py

```diff
@@ -425,8 +435,15 @@ def add_new_agent(self, id, x, y, orient, with_proove=False):
     def create_agents(self):
         """Creating agents according to how the simulation class was initialized"""
         for i in range(self.N):
-            x = np.random.randint(self.agent_radii, self.WIDTH - self.agent_radii)
-            y = np.random.randint(self.agent_radii, self.HEIGHT - self.agent_radii)
+
+            # allowing agents to overlap arena borders (maximum overlap is radius of patch)
+            x = np.random.randint(self.window_pad - self.agent_radii, self.WIDTH + self.window_pad - self.agent_radii)
+            y = np.random.randint(self.window_pad - self.agent_radii, self.HEIGHT + self.window_pad - self.agent_radii)
+
+            # for inhibiting agents to overlap arena borders comment the previous two lines  and uncomment following two lines
+            # x = np.random.randint(self.window_pad, self.WIDTH + self.window_pad - 2 * self.agent_radii)
+            # y = np.random.randint(self.window_pad, self.HEIGHT + self.window_pad - 2 * self.agent_radii)
+
```

Comment:
```suggestion
```
As agents will move it doesn't really matter if they overlap or not. To keep the code quality, I suggest to remove this  comment section